extensions/notebook/package.json

```diff
@@ -667,7 +667,7 @@
     "js-yaml": "^3.12.1",
     "node-fetch": "^2.6.1",
     "request": "^2.88.0",
-    "tar": "^6.0.1",
+    "tar": "^6.1.1",
```

Comment:
Should bump this to 6.1.6 as well - especially since it's already set as that in the yarn.lock

Reply:
ok, will do.  I just grabbed the version mentioned in the alert.